src/expr/dtype.h

```diff
@@ -489,6 +499,47 @@ class DType
    * Helper for mkGroundTerm and mkGroundValue above.
    */
   Node mkGroundTermInternal(TypeNode t, bool isValue) const;
+  /**
+   * This method is used to get strict subfield types of this datatype.
+   *
+   * A subfield type T of a datatype type D is a type such that a value of
```

Comment:
```suggestion
   * A subfield type T of a datatype type D is a type such that a term of
```

?

Reply:
Value is more precise here. Notice that values of datatypes have term structure.  We don't want to say "term" here since this would include arbitrary ways of constructing terms of datatype type.